docs/getting_started/ibexa_cloud_guide.md

```diff
@@ -0,0 +1,119 @@
+---
+description: The Ibexa Cloud product guide provides a full description of its features as well as the benefits it brings to the client.
+---
+
+# [[= product_name_cloud =]] product guide
+
+## What is [[= product_name_cloud =]]
+
+[[= product_name_cloud =]] is a cloud hosting platform that enables you to host your application in the cloud by using the [Platform.sh](https://platform.sh/) service.
+It also establishes the framework for the potential growth and improves project delivery.
+As a diverse Platform as a Service (PaaS), it's designed to allow you to focus on the crucial things.
+
+[[= product_name_cloud =]] is a part of [[= product_name =]] - a software that is designed to provide your business with all the features, functionality, and support your need to transform your business for the digital age.
+
+![Ibexa Cloud - part of Ibexa DXP](ibexa_cloud_dxp.png)
+
+## Availability
+
+[[= product_name_cloud =]] is available in all [[= product_name =]] editions.
+
+## How does [[= product_name_cloud =]] work
+
+[[= product_name_cloud =]], as a hosting platform, is designed to streamline the development and testing processes, allowing you to deliver new features much faster.
+It ensures that developers dedicate more time to important development activities rather than maintaining databases, queues, search engines, and operating systems.
+It shifts the focus away from fundamental operations and toward improving your digital services with additional features, and functionality.
+
+![How does Ibexa Cloud work](ibexa_cloud.png)
+
+## [[= product_name_cloud =]] account
+
+If you want to use [[= product_name_cloud =]], you need to follow [installation process](install_on_ibexa_cloud.md) and check the [requirements](requirements.md#ibexa-cloud-requirements-and-setup).
+
+To use [[= product_name_cloud =]], you must make arrangements with [[= product_name_base =]] to get and set up a user account.
+To do so, contact your Partner Manager, or Sales representatives team, or fill out the form available at this link: https://www.ibexa.co/about-ibexa/contact-us.
+
+When you have an account, you can log in to https://console.ibexa.cloud.
+
+## Capabilities
+
+### Platform as a Service
+
+[[= product_name_cloud =]] is a PaaS provider. It's a cloud-based subscription service that you can use for developing, managing, and running applications without infrastructure concerns.
+This cloud computing approach gives users access to a full cloud platform, including hardware, software, and infrastructure.
+PaaS eliminates the requirement to buy and install the necessary hardware and software. All you have to do is access it and you can start deploying resources and developing right away.
+
+![How PaaS works](how_paas_works.png)
+
```

Comment:
I think we might miss the Observability Suite https://platform.sh/features/observability-suite/
It's included in each subscription